### PR TITLE
Improve DatasetDetail component

### PR DESCRIPTION
**Pull request by tmerlier**
- Translation
- Display catalogs name
- OrganizationsSection
- Lineage

### CODE REVIEW

**Comment by tmerlier** [comment]
Instead of getting all the catalogs and then sort them in `DatasetDetail`. We can fetch one by one directly in `CatalogsSection`.

File: src/components/Dataset/CatalogsSection.js [hunk 1]
```diff
@@ -10,15 +10,18 @@ const CatalogsSection = ({catalogs}) => {
     catalogs: {
       display: 'flex',
       flexDirection: 'column',
+    },
+    link: {
+      cursor: 'pointer',
```
Comment (jdesboeufs) [inline src/components/Dataset/CatalogsSection.js]: Is it required?

File: src/components/Dataset/DatasetDetail.js [hunk 2]
```diff
@@ -26,8 +29,13 @@ export default class DatasetDetail extends Component {
     return waitForDataAndSetState(fetchDataset(this.props.params.datasetId), this, 'dataset')
   }
 
+  updateCatalogs() {
+    return waitForDataAndSetState(fetchCatalogs(this.props.params.datasetId), this, 'catalogs')
```
Comment (jdesboeufs) [inline src/components/Dataset/DatasetDetail.js]: `datasetId` is not required here

File: src/components/Dataset/DatasetDetail.js [hunk 3]
```diff
@@ -45,14 +53,17 @@ export default class DatasetDetail extends Component {
...
       <div style={styles.container}>
         <DatasetSection dataset={dataset} />
         <div style={styles.sections}>
-          <DownloadsSection links={dataset.metadata.links} />
+          <LinksSection links={dataset.metadata.links} />
           <KeywordsSection keywords={dataset.metadata.keywords} />
-          <CatalogsSection catalogs={dataset.catalogs} />
+          <OrganizationsSection organizations={dataset.organizations} />
+          <CatalogsSection catalogs={catalogs.filter(catalog => {
+              return dataset.catalogs.includes(catalog.id)
```
Comment (jdesboeufs) [inline src/components/Dataset/DatasetDetail.js]: You can use a simple one-liner (drop return instruction)

File: src/components/Dataset/DatasetSection.js [hunk 4]
```diff
@@ -21,10 +21,11 @@ const DatasetSection = ({dataset}) => {
       <div style={styles.infos} className="infos">
         <div>Type : <span>{type}</span></div>
         <div>Licence : <span>{license}</span></div>
-        <div>Dernière mise à jour de la fiche : <span>{revisionDate}</span></div>
+        <div>Last update : <span>{revisionDate}</span></div>
       </div>
-      <div>Identifiant technique de la fiche : <span>{dataset.metadata.id}</span></div>
-      <h5>{dataset.metadata.description}</h5>
+      <div>Origin of data : <span>{dataset.metadata.lineage}</span></div>
```
Comment (jdesboeufs) [inline src/components/Dataset/DatasetSection.js]: Ugly and wrong place :p

File: src/components/Dataset/LinksSection.js [hunk 5]
```diff
@@ -9,16 +9,19 @@ const DownloadsSection = ({links}) => {
     links: {
       display: 'flex',
       flexDirection: 'column',
+    },
+    link: {
```
Comment (jdesboeufs) [inline src/components/Dataset/LinksSection.js]: Same question

**Review by jdesboeufs — CHANGES_REQUESTED** [review]

File: src/components/Dataset/CatalogsSection.js (hunk 1 above)
Comment (tmerlier) [reply src/components/Dataset/CatalogsSection.js]: Yes if not the cursor does not change

**Comment by tmerlier** [comment]
@jdesboeufs

**Review by jdesboeufs — APPROVED** [review]

**Pull request merged**